app/models/comment.rb

```diff
@@ -1,2 +1,20 @@
 class Comment < ApplicationRecord
+  belongs_to :user
+  belongs_to :post
+
+  after_create :update_comments_counter_on_create
+  after_destroy :update_comments_counter_on_destroy
```

Comment:
- [ ] It would good to add `after_save` to call the `#update_comments_counter` method every time a new post has been saved to the database,

```
`after_save :update_comments_counter`
```